spacy/tests/lang/es/test_exception.py

```diff
@@ -1,6 +1,16 @@
 import pytest
 
 
+@pytest.mark.issue(3277)
+def test_issue3277(es_tokenizer):
+    """Test that hyphens are split correctly as prefixes."""
+    doc = es_tokenizer("—Yo me llamo... –murmuró el niño– Emilio Sánchez Pérez.")
+    assert len(doc) == 14
+    assert doc[0].text == "\u2014"
+    assert doc[5].text == "\u2013"
+    assert doc[9].text == "\u2013"
+
+
```

Comment:
Here as well, this test is more specific to the tokenization in general rather than to `tokenizer_exceptions.py`